infra/lib/fhir-server-stack.ts

```diff
@@ -385,6 +403,21 @@ export class FHIRServerStack extends Stack {
       evaluationPeriods: 1,
       treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
     });
+
+    /**
+     * For Aurora Serverless, this alarm is not important as it auto-scales. However, we always
+     * create this alarm because of compliance controls (SOC2).
+     * @see: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Overview.StorageReliability.html#aurora-storage-growth
+     */
+    createAlarm({
+      metric: dbCluster.metricFreeLocalStorage(),
+      name: "FreeLocalStorageAlarm",
+      threshold: mbToBytes(10_000),
```

Comment:
nit: magic number

Reply:
I see your point, but this is really just some number, see the comment above it:

> For Aurora Serverless, this alarm is not important as it auto-scales. However, we always create this alarm because of compliance controls (SOC2).